### PR TITLE
Added Testimonials for Intraserve and HPE Labs

### LINKED ISSUE #501: [NG] Add Testimonial from HPE Labs, Cloudless
**Current Behavior**
"HPE Labs, Cloudless" gave a glowing review of their [adoption of Meshery](https://github.com/layer5io/meshery/blob/master/ADOPTERS.md).

**Desired Behavior**
This testimony can be added to the layer5-ng Testimony component.

### PR DESCRIPTION
**Pull request by Jashpatel1**
**Description**

This PR fixes #501 #502   

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/assets/data/testimonial/index.js [hunk 1]
```diff
@@ -10,16 +10,16 @@ const data = {
...
+      dsignation: 'Intraserve Systems',
       thumb: image1
     },
     {
         quote:
-          'This app is on one assistant statement caught agency, the atprior from peace counter. A to is home, between history; With the headline is an the narrow size part. In ride ourselves,you to out',
-        author: 'Smith Doe',
-        dsignation: 'Marketer',
+          'Our Complex, Dynamic, and Intelligent system demands leverage different tools to efficiently manage service exchange and ecosystem at Global Scale; for instance, we need to identify right fit Service Mesh to address varying workload scenarios, distributed systems across locations, different security tiers, minimal latency etc. Meshery helps us curate Service Mesh for tailored need by helping us running simulations and identifying the right fit Service Mesh that works efficiently across diverse ecosystem of tools.',
+        author: 'Cloudless',
```
Comment (Tanuj22) [inline src/assets/data/testimonial/index.js:21]: Is the author supposed to be Cloudless? @leecalcote

Comment (leecalcote) [reply src/assets/data/testimonial/index.js:21]: @Tanuj22 Good question. No, it’s not. Cloudless is a division of HPE Labs. 

For now, we can use “Distinguished Technologist” as the author.

File: src/assets/data/testimonial/index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import image1 from '../../images/app/testimonial/testimonial-thumb-1.jpg';
-import image2 from '../../images/app/testimonial/testimonial-thumb-2.jpg';
+import image1 from '../../images/app/testimonial/intraserve.png';
+import image2 from '../../images/app/testimonial/hpe.png';
```
Comment (leecalcote) [inline src/assets/data/testimonial/index.js:2]: In general, can we try to use SVGs where possible? I understand that they’re likely isn’t one available for intraserve, but for HPE, You can find one in the current Layer5 site.

Comment (Jashpatel1) [reply src/assets/data/testimonial/index.js:2]: @leecalcote The one present in layer5 repo is a different type of logo, which has the entire full form of HPE(making it rectangular), while we want a square or circle one(also on Internet SVG is available for the rectangular one). So i chose to go with this one.

Comment (leecalcote) [reply src/assets/data/testimonial/index.js:2]: Ok, I'll find one. Thanks, @Jashpatel1

**Pull request merged**